app-header-layout/demo/demo1.html

```diff
@@ -52,10 +53,10 @@
   <!-- default to use body scroller -->
   <app-header-layout>
 
-    <app-header condenses reveals>
+    <app-header effects="waterfall" condenses reveals>
       <app-toolbar>
         <paper-icon-button icon="menu"></paper-icon-button>
-        <div class="flex"></div>
+        <div title></div>
```

Comment:
Can you also remove the [classes/iron-flex-layout.html import](https://github.com/PolymerLabs/app-layout/blob/header-snap/app-header-layout/demo/demo1.html#L22)?